openedx/features/partners/forms.py

```diff
@@ -114,8 +114,8 @@ def clean_registration_data(self, registration_data):
         registration_data.update(self.cleaned_data)
 
     def clean_country(self):
-        country = self.cleaned_data['country']
-        cleaned_country = next((code for code, label in COUNTRIES.items() if label.lower() == country.lower()), None)
+        country_code = self.cleaned_data['country']
+        cleaned_country = next((code for code in COUNTRIES.keys() if code == country_code), None)
```

Comment:
@abdullah998 just a suggestion it is not mandatory to implement, but what if it is written as below.
```
        if country_code in COUNTRIES.keys():
            return country_code
        raise ValidationError(_('Please select country.'))
```

Reply:
Looks good! Thanks for this.